### DISCUSSION TITLE
Use OpenEBS on a mounted glusterfs volume

### DISCUSSION THREAD

**Discussion by urbaman** [openebs/openebs#3535] [Q&A] [answered]
Hi,

I could mount a glusterfs volume on all my microk8s nodes (masters and workers).
Can I then define a Jive volume/storageclass on that mount point, or it needs to be only ext4/xfs?

Thank you.

**Comment by avishnu** [comment, marked as answer]
Did you try with Jiva as per your query?

**Comment by orville-wright** [comment]
Hi @mohammada315 
Please note: the openEBS Jiva project has been ``` archived ``` and is no longer a core active component of openEBS.
- Please see ``` Project announcement ``` here: https://github.com/openebs/openebs/issues/3709

- the repo and all dependencies have been migrated to the CNCF owned [openEBS Archive](https://github.com/openebs-archive)
- Support for Jiva will be done on a ```best effort``` basis cvia the CNCF openEBS Slack channel

We recommend users migrate their Jiva deployments to other OpenEBS Storage Engines.
